### ISSUE TITLE
Get rid of deprecated ConnectedMulticastPort and ConnectedPort

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3992]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#3992|
|Type|task|
|Created|2009-08-31 16:08:27Z|
|Branch|https://github.com/twisted/twisted/tree/remove-ConnectedMulticastPort-and-ConnectedPort-3992|

In [twisted/internet/udp.py](twisted/internet/udp.py) the ```ConnectedMulticastPort``` and ```ConnectedPort``` classes are deprecated in what seems an old-fashioned way, which indicates to me that they've been there for years and it's about time to get rid of it.

<details><summary>Searchable metadata</summary>

```
trac-id__3992 3992
type__task task
reporter__thijs thijs
priority__normal normal
milestone__ 
branch__branches_remove_ConnectedMulticastPort_and_ConnectedPort_3992 branches/remove-ConnectedMulticastPort-and-ConnectedPort-3992
branch_author__cyli cyli
status__closed closed
resolution__fixed fixed
component__core core
keywords__easy easy
time__1251734907000000 1251734907000000
changetime__1274588265000000 1274588265000000
version__None None
owner__ 
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_1"></a>|@thijstriemstra commented|
|-|-|

Also adding ```ConnectedPort``` to the ticket description.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)|@glyph removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_3"></a>|@cyli commented|
|-|-|

(In [28988]) Branching to 'remove-ConnectedMulticastPort-and-ConnectedPort-3992'

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_4"></a>|@cyli set status to `closed`|
|-|-|

This fix was rolled into [#4075](https://github.com/twisted/twisted/issues/4075), in r27686.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
